latch_cli/snakemake/workflow.py

```diff
@@ -298,12 +298,14 @@ def __init__(self, cache_tasks: bool = False):
         display_name = metadata._snakemake_metadata.display_name
         name = metadata._snakemake_metadata.name
 
+        inputs = {k: (v.type, v.default) for k, v in parameter_metadata.items()}
+        inputs["dry_run"] = (bool, False)
```

Comment:
we definitely shouldn't overwrite user defined parameters (like what happens if the user's workflow has a parameter called `dry_run` lol)

you should signal a dry run by using an environment variable or something instead